README.md

```diff
@@ -43,55 +43,54 @@ The following examples assume you are using a kotlin build script ([which are mu
     }
     ```
 
-### Features
-
-- You can automatically add Mindustry dependencies with :
+3. Automatically add Mindustry dependencies with :
 
-  ```kotlin
-  import fr.xpdustry.toxopid.util.anukenJitpack
-  import fr.xpdustry.toxopid.util.mindustryDependencies
-
-  repositories {
-      mavenCentral()
-      anukenJitpack()
-  }
+    ```kotlin
+    import fr.xpdustry.toxopid.util.anukenJitpack
+    import fr.xpdustry.toxopid.util.mindustryDependencies
 
-  dependencies {
-      mindustryDependencies()
-  }
-  ```
+    repositories {
+        mavenCentral()
+        anukenJitpack()
+    }
 
-- You can use the info of your `[mod|plugin].[h]json` in your build script with `ModMetadata` :
+    dependencies {
+        mindustryDependencies()
+    }
+    ```
 
-  ```kotlin
-  import fr.xpdustry.toxopid.util.ModMetadata
-  
-  val metadata = ModMetadata.fromJson(project.file("mod.hjson"))
-  // Setting the project version from the one located in "mod.json"
-  project.version = metadata.version
-  ```
+4.  Load the info of your `[mod|plugin].[h]json` in your build script with `ModMetadata` :
+    ```kotlin
+    import fr.xpdustry.toxopid.util.ModMetadata
 
-  or directly generating your `[mod|plugin].[h]json` inside the build script and include it in the jar :
+    val metadata = ModMetadata.fromJson(project.file("mod.hjson"))
```

Comment:
```suggestion
    val metadata = ModMetadata.fromJson(project.file("mod.json"))
```
Consistent with the comment below (and also because I tend to avoid hjson).